functions/backup.sh

```diff
@@ -133,14 +133,13 @@ amanda_setup() {
   fi
 
   /usr/sbin/exim --version 2>/dev/null >/dev/null
+  local adminmail="root@$(/bin/hostname)"
   if [ $? -ne 0 ]; then
      if [ -n "$INTERACTIVE" ]; then
         if (whiptail --title "No exim mail transfer agent" --yes-button "Install EXIM4" --no-button "MTA already exist, ignore installation" --defaultyes --yesno "Seems exim is not installed as a mail transfer agent.\nAmanda needs one to be able to send emails.\nOnly choose to ignore if you know there's a working mail transfer agent other than exim on your system.\nDo you want to continue with EXIM4 installation ?" 15 80) then
            exim_setup
         fi
-        local adminmail=$(whiptail --title "Admin reports" --inputbox "Enter the email address to send backup reports to." 10 60 3>&1 1>&2 2>&3)
-     else
-        local adminmail="root@$(/bin/hostname)"
+        adminmail=$(whiptail --title "Admin reports" --inputbox "Enter the email address to send backup reports to." 10 60 3>&1 1>&2 2>&3)
```

Comment:
Thought I'd write my comment here too, should the user be able to set `adminmail` if exim already existed?

Reply:
yes he should.
See changed code. That now will also avoid the error to happen if a user enters an empty mail address.